runtime/foucoco/src/lib.rs

```diff
@@ -1195,6 +1536,7 @@ construct_runtime!(
 		VaultRegistry: vault_registry::{Pallet, Call, Config<T>, Storage, Event<T>, ValidateUnsigned} = 69,
 		VaultRewards: reward::{Pallet, Call, Storage, Event<T>} = 70,
 		VaultStaking: staking::{Pallet, Storage, Event<T>} = 71,
+		TokenAllowance: orml_currencies_allowance_ext::{Pallet, Storage, Call, Event<T>} = 72,
```

Comment:
Should we maybe set a different number and add a new line above this? It now looks like it belongs to the spacewalk pallets while it is not necessarily related to that. Maybe change to
```
VaultStaking: staking::{Pallet, Storage, Event<T>} = 71,

TokenAllowance: orml_currencies_allowance_ext::{Pallet, Storage, Call, Event<T>} = 80,
```

Reply:
Done